src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationItem.cs

```diff
@@ -344,11 +239,6 @@ internal void LoadElementExtensions(XmlReader readerOverUnparsedExtensions, int
             _extensions.LoadElementExtensions(readerOverUnparsedExtensions, maxExtensionSize);
         }
 
-        internal void LoadElementExtensions(XmlBuffer buffer)
-        {
-            _extensions.LoadElementExtensions(buffer);
-        }
+        internal void LoadElementExtensions(XmlBuffer buffer) => _extensions.LoadElementExtensions(buffer);
     }
```

Comment:
Inconsistent conversion of methods to delegate-based. The other LoadElementExtensions overload plus WriteElementExtensions and WriteAttributeExtensions.